src/odemis/gui/cont/acquisition/cryo_acq.py

```diff
@@ -349,6 +381,15 @@ def _on_feature_acquisition_done(self, future):
         except Exception as e:
             logging.warning(f"Error resetting acquisition GUI: {e}")
 
+    def _get_selected_features(self):
+        """Get the selected features from the checklist."""
+        features = [
+            f
+            for i, f in enumerate(self._tab_data.main.features.value)
+            if self._panel.acquire_features_chk_list.IsChecked(i)
+        ]
```

Comment:
You could use the [`CheckedItems`](https://docs.wxpython.org/wx.CheckBox.html#wx.CheckBox.IsChecked) prop to simplify. Something like:
```suggestion
           features = [self._tab_data.main.features.value[i] for i in self._panel.acquire_features_chk_list.CheckedItems]
```